inference-engine/tests/functional/plugin/cpu/shared_tests_instances/single_layer_tests/activation.cpp

```diff
@@ -23,34 +23,36 @@ const std::vector<InferenceEngine::Precision> netPrecisions = {
 };
 
 const std::map<ActivationTypes, std::vector<std::vector<float>>> activationTypes = {
-        {Sigmoid,     {}},
-        {Tanh,        {}},
-        {Relu,        {}},
-        {Exp,         {}},
-        {Log,         {}},
-        {Sign,        {}},
-        {Abs,         {}},
-        {Clamp,       {{-2.0f, 2.0f}}},
-        {Negative,    {}},
-        {Acos,        {}},
-        {Asin,        {}},
-        {Atan,        {}},
-        {Cos,         {}},
-        {Cosh,        {}},
-        {Floor,       {}},
-        {Sin,         {}},
-        {Sinh,        {}},
-        {Sqrt,        {}},
-        {Tan,         {}},
-        {Elu,         {{0.1f}}},
-        {Erf,         {}},
-        {HardSigmoid, {{0.2f, 0.5f}}},
-        {Selu,        {{1.6732f, 1.0507f}}},
-        {Ceiling,     {}},
-        {Mish,        {}},
-        {HSwish,      {}},
-        {SoftPlus,    {}},
-        {HSigmoid,    {}}
+        {Sigmoid,               {}},
+        {Tanh,                  {}},
+        {Relu,                  {}},
+        {Exp,                   {}},
+        {Log,                   {}},
+        {Sign,                  {}},
+        {Abs,                   {}},
+        {Clamp,                 {{-2.0f, 2.0f}}},
+        {Negative,              {}},
+        {Acos,                  {}},
+        {Asin,                  {}},
+        {Atan,                  {}},
+        {Cos,                   {}},
+        {Cosh,                  {}},
+        {Floor,                 {}},
+        {Sin,                   {}},
+        {Sinh,                  {}},
+        {Sqrt,                  {}},
+        {Tan,                   {}},
+        {Elu,                   {{0.1f}}},
+        {Erf,                   {}},
+        {HardSigmoid,           {{0.2f, 0.5f}}},
+        {Selu,                  {{1.6732f, 1.0507f}}},
+        {Ceiling,               {}},
+        {Mish,                  {}},
+        {HSwish,                {}},
+        {SoftPlus,              {}},
+        {HSigmoid,              {}},
+        {RoundHalfToEven,       {}},
```

Comment:
Could you add cpu instance please? (`RoundHalfToEven`, `RoundHalfAwayFromZero`)

Reply:
Do you mean
`INSTANTIATE_TEST_CASE_P(smoke_Activation_Basic_Round, ActivationLayerTest, basicRoundCases, ActivationLayerTest::getTestCaseName);`
with separate `activationRoundTypes` and `basicRoundCases` like cpu instance for Prelu?

Reply:
Ooh, sorry, my bad ;) I missed `cpu` in the flename